apps/web/components/sections/home/hero.tsx

```
@@ -0,0 +1,112 @@
+
+import { useCallback } from 'react';
+import { Badge } from '~/components/base/badge';
+import { Button } from '~/components/base/button';
+import { badgeVariants, staggerChildren } from '~/lib/constants/animations';
+import { categories } from '~/lib/mock-data/mock-hero-section';
+import type { Category } from '~/lib/types/home.types';
+
+export function Hero() {
+  const shouldReduceMotion = useReducedMotion();
+
+  const renderCategory = useCallback((category: Category) => (
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: shouldReduceMotion

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)